docs/monitoring/08-01-observe-application-metrics.md

```diff
@@ -0,0 +1,122 @@
+---
+title: Observe application metrics
+type: Tutorials
+---
+
+This tutorial shows how you can check the list and changing values of all metrics exposed by a sample Go service by redirecting the metrics port and the default Prometheus server port to a localhost.
```

Comment:
as these are metrics that are exposed instead of values, I would modify it a bit into sth like:
```
This tutorial shows how you can observe your application metrics. Learn how to list all metrics exposed by a sample Go service and see their changing values by redirecting the metrics port and the default Prometheus server port to a localhost.
```